src/components/JobCard/JobCard.tsx

```diff
@@ -26,21 +37,37 @@ function JobCard({
   setPendingJobs,
   filteredJobs,
   setFilteredJobs,
+  bookmarkedJobs,
+  setBookmarkedJobs,
 }: JobCardProps) {
+  const { userObject } = useContext(AuthContext);
+  const [bookmarkedValue, setBookmarked] = useState<boolean>();
+  const userObjectToString = JSON.stringify(userObject?.bookmarkedJobs);
+
+  useEffect(() => {
+    const getBookmarked = () => {
+      const bookmarks = getBookmarks(job.id, userObject);
```

Comment:
I think you can remove the userObject as an argument to the getBookmarks function. instead i might say to get the userObject in the user.ts file and use that with the function

Reply:
usecontext can only be called within a react component/screen so i don't think i can get the userObject in the user.ts file